src/sentry/event_manager.py

```diff
@@ -777,21 +777,8 @@ def save(self, project_id, raw=False, assume_normalized=False):
 
         # save the event unless its been sampled
         if not is_sample:
-            try:
-                with transaction.atomic(using=router.db_for_write(Event)):
-                    event.save()
-            except IntegrityError:
-                logger.info(
-                    "duplicate.found",
-                    exc_info=True,
-                    extra={
-                        "event_uuid": event_id,
-                        "project_id": project.id,
-                        "group_id": group.id if group else None,
-                        "model": Event.__name__,
-                    },
-                )
-                return event
+            to_write = dict(data.items())
+            nodestore.set(event_id, to_write)
```

Comment:
I don't think this is right. The node id should be something like:
> md5("{}:{}".format(project_id, event_id)).hexdigest()

but please verify first that it is the way we save them today.

Also why going directly to nodestore skipping NodeData ?

Reply:
Oops, thanks. I don't think we need any of the functionality from NodeData. Unless there's a reason we want to avoid calling nodestore.set directly.